consensus/state.go

```diff
@@ -1554,39 +1554,41 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 		var (
 			commitSize = block.LastCommit.Size()
 			valSetLen  = len(cs.LastValidators.Validators)
+			address types.Address = types.Address{}
```

Comment:
```suggestion
			address types.Address
```

Reply:
what's the standard here; fixes in the PR and squash on merge, or update my branch to have a single squashed commit?

Reply:
fixes in PR and then we squash on merge